<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comparação</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>10</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N. Comparacoes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N. Trocas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempo(us)</string>
    </property>
   </widget>
   <widget class="QLabel" name="cocktailName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Cocktail</string>
    </property>
   </widget>
   <widget class="QLabel" name="cocktailCompare">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>50</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cocktailChanges">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cocktailTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectionName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectionCompare">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>80</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectionChanges">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectionTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="insertionName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertion</string>
    </property>
   </widget>
   <widget class="QLabel" name="insertionCompare">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>110</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="insertionChanges">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="insertionTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="bubbleName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Bubble</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>140</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="shellName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Shell</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>170</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mergeName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>200</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="quickName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>230</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>400</width>
      <height>250</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>250</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTamanho_Vetor">
    <property name="title">
     <string>Tamanho Vetor</string>
    </property>
    <addaction name="action20_Posicoes"/>
    <addaction name="action50_Posicoes"/>
    <addaction name="action100_Posicoes"/>
    <addaction name="actionDigite_o_valor_desejado"/>
   </widget>
   <widget class="QMenu" name="menuPrimeiro_Metodo">
    <property name="title">
     <string>Metodo de Ordenação</string>
    </property>
    <addaction name="actionCocktail"/>
    <addaction name="actionSelection"/>
    <addaction name="actionInsertion"/>
    <addaction name="actionBubble"/>
    <addaction name="actionShell"/>
    <addaction name="actionMerge"/>
    <addaction name="actionQuick"/>
   </widget>
   <widget class="QMenu" name="menuSegundo_Metodo">
    <property name="title">
     <string>Gráficos</string>
    </property>
    <addaction name="actionCocktail_2"/>
    <addaction name="actionSelection_2"/>
    <addaction name="actionInsertion_2"/>
    <addaction name="actionBubble_2"/>
    <addaction name="actionShell_2"/>
    <addaction name="actionMerge_2"/>
    <addaction name="actionQuick_2"/>
   </widget>
   <addaction name="menuTamanho_Vetor"/>
   <addaction name="menuPrimeiro_Metodo"/>
   <addaction name="menuSegundo_Metodo"/>
  </widget>
  <action name="action20_Posicoes">
   <property name="text">
    <string>20 Posicoes</string>
   </property>
  </action>
  <action name="action50_Posicoes">
   <property name="text">
    <string>50 Posicoes</string>
   </property>
  </action>
  <action name="action100_Posicoes">
   <property name="text">
    <string>100 Posicoes</string>
   </property>
  </action>
  <action name="actionCocktail">
   <property name="text">
    <string>Cocktail</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionInsertion">
   <property name="text">
    <string>Insertion</string>
   </property>
  </action>
  <action name="actionBubble">
   <property name="text">
    <string>Bubble</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="actionQuick">
   <property name="text">
    <string>Quick</string>
   </property>
  </action>
  <action name="actionShell">
   <property name="text">
    <string>Shell</string>
   </property>
  </action>
  <action name="actionCocktail_2">
   <property name="text">
    <string>Cocktail</string>
   </property>
  </action>
  <action name="actionSelection_2">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionInsertion_2">
   <property name="text">
    <string>Insertion</string>
   </property>
  </action>
  <action name="actionBubble_2">
   <property name="text">
    <string>Bubble</string>
   </property>
  </action>
  <action name="actionShell_2">
   <property name="text">
    <string>Shell</string>
   </property>
  </action>
  <action name="actionMerge_2">
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="actionQuick_2">
   <property name="text">
    <string>Quick</string>
   </property>
  </action>
  <action name="actionDigite_o_valor_desejado">
   <property name="text">
    <string>Digite o valor desejado</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
